website/src/pages/Blog/Author/index.tsx

```diff
@@ -89,56 +87,56 @@ const Blog: React.FC = () => {
   };
 
   useEffect(() => {
-    if(filteredAuthorData.length && currentOrigin && authorName) {
-      const authorData = filteredAuthorData.find(res => res.author.toLowerCase().replace(/[^\w ]+/g,'').replace(/ +/g,'-') === authorName);
-      const { author, author_info } = authorData!;
-      const image = `${currentOrigin}/images/blog/authors/${getAvatar(filteredAuthorData[0]?.author)}.png`;
+    if(authorBlogsData?.length && currentOrigin && authorName) {
+      const { author, author_info } = authorBlogsData[0]!;
+      const image = `${currentOrigin}/images/blog/authors/${getAvatar(authorBlogsData[0]?.author)}.png`;
       const url = `${currentOrigin}/blog/author/${authorName}`;
       setAuthorMetadata({ author, author_info, image, url });
     }
-  }, [filteredAuthorData, currentOrigin, authorName]);
+  }, [authorBlogsData, currentOrigin, authorName]);
 
   return (
     <>
      { authorMetadata && (
        <Metadata title={authorMetadata?.author} description={authorMetadata?.author_info} url={authorMetadata?.url} image={authorMetadata?.image} isPost={false} type={METADATA_TYPES.SERIES}  />
        )}
-        <>
+        { authorBlogsData && (
+          <>
           <div className={classes.root}>
             <Container className={classes.sectionDiv}>
-              {(width > mobileBreakpoint && filteredAuthorData.length > 0) &&
+              {(width > mobileBreakpoint && authorBlogsData.length > 0) &&
                 <Breadcrumbs aria-label="breadcrumb" className={classes.breadCrumbs}>
                   <Link color="inherit" href="/blog">
                     {t('blog.blog')}
                   </Link>
                   <Link color="inherit" href={`/blog/author/${authorName}`}>
-                      {filteredAuthorData[0]?.author}
+                      {authorBlogsData[0]?.author}
                   </Link>
                 </Breadcrumbs>
               }
               <div className={classes.authorWrapper}>
                 <Avatar
-                  alt={filteredAuthorData[0]?.author}
-                  src={`/images/blog/authors/${getAvatar(filteredAuthorData[0]?.author)}.png`}
+                  alt={authorBlogsData[0]?.author}
+                  src={`/images/blog/authors/${getAvatar(authorBlogsData[0]?.author)}.png`}
                   className={classes.large}
                 />
-                <div>
-                  <h1 className={classes.authorText}>{filteredAuthorData[0]?.author || authorName}</h1>
-                  <p className={classes.authorDesc}>{filteredAuthorData[0]?.author_info}</p>
-                </div>
```

Comment:
@IsAmrish this section needs to be like this

Reply:
Thankyou!